.gitignore

```diff
@@ -1 +1,71 @@
-.discourse-site
\ No newline at end of file
+.discourse-site
```

Comment:
Sorry to keep pushing back on this @rogercreagh, but this `.gitignore` file showed up again...